portfolio/src/main/webapp/scripts/comments.js

```diff
@@ -114,7 +114,7 @@ function scoreToColor(score) {
  */
 function scoreToIndex(score) {
   let index = Math.floor((score+1)*2.5);
-  index = min(4, index);
+  index = Math.min(4, index);
```

Comment:
[Optional] This should probably be part of a separate PR but since it's so small I think it's ok to leave it in here if you don't want the hassle of splitting it out.

Reply:
Ok, thanks